src/matrix/wishart.jl

```diff
@@ -151,7 +151,7 @@ function _wishart_genA!(rng::AbstractRNG, p::Int, df::Real, A::AbstractMatrix)
     #               ~ Normal()                  when i > j
     #
     A .= zero(eltype(A))
-    for i = 1:p
+    for i in 1:p
```

Comment:
@matbesancon Changes like these show up in my diff.

Reply:
Yes this was added you're right